src/pretix/presale/views/event.py

```diff
@@ -348,6 +348,9 @@ def get(self, request, *args, **kwargs):
             r._csp_ignore = True
             return r
 
+        if 'web' not in request.event.sales_channels:
```

Comment:
Maybe better use ``request.sales_channel.identifier`` in case a middleware modifies that?